dev/apollo-federation/router.yaml

```diff
@@ -7,8 +7,20 @@ supergraph:
   listen: 0.0.0.0:4004
   introspection: true
   path: /graphql
+cors:
+  match_origins:
+    - "^http://([a-z0-9]+[.])*" # http
+    - "^https://([a-z0-9]+[.])*" # https
+  allow_credentials: true # for cookie auth
```

Comment:
suggested value for prod:

cors:
  origins:
    - https://studio.apollographql.com # Keep this so Apollo Studio can still run queries against your router
  match_origins:
    - "^https://([a-z0-9]+[.])*blink[.]sv$"
  allow_credentials: true # for cookie auth